### PR TITLE
chore(Badge/MenuToggle): updated styling

### LINKED ISSUE #6463: Penta Badge/Menu Toggle - update styling when only badge is passed to Menu toggle
**Describe the enhancement or change**
Followup to https://github.com/patternfly/patternfly/pull/6430, which contains followup change to be made/looked into

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**

### PR DESCRIPTION
**Pull request by thatblindgeye**
Closes #6463 

Currently I'm applying some additional styling to a `pf-m-small` MenuToggle rather than only when it's inside a breadcrumb dropdown. I can update to only change that styling when the MenuToggle is inside the breadcrumb dropdown, but was curious if it would make sense to adjust other styling for a `pf-m-small` Menu Toggle rather than only the top/bottom padding.

Also note that the `pf-m-text` modifier on MenuToggle no longer actually does anything, but we've kept it on the React component.

### CODE REVIEW

**Comment by andrew-ronaldson** [comment]
The menu toggle example looks good here
Screenshot 2024-04-30 at 12 52 48 PM

The breadcrumb example crops the hover/focus states to fit
Screenshot 2024-04-30 at 12 53 44 PM 

is this caused by the breadcrumbs height?

**Comment by thatblindgeye** [comment]
@andrew-ronaldson that's from setting the padding top/bottom on the MenuToggle in that particular instance to 0 to try and get it closer to being contained in the outer breadcrumb element. I can remove those style overrides which would result in the following:

image
image

**Comment by thatblindgeye** [comment]
@andrew-ronaldson Does this look more inline with what we'd want?

image

I updated the `--#{$breadcrumb}__dropdown--MarginInline[Start|End]` var to an xs spacer, instead of it creating a negative margin based on `calc(var(--#{$breadcrumb}__item--MarginInlineEnd) * -1)`

**Comment by andrew-ronaldson** [comment]
Looks good with that update @thatblindgeye

**Review by andrew-ronaldson — APPROVED** [review]
looks dope!

**Review by srambach — APPROVED** [review]
🏅 Looks good to me.

File: src/patternfly/components/Breadcrumb/breadcrumb.scss [hunk 1]
```diff
@@ -116,6 +118,8 @@
   margin-inline-end: var(--#{$breadcrumb}__menu-toggle--MarginInlineEnd);
 
   .#{$menu-toggle} {
+    padding-block-start: var(--#{$breadcrumb}__menu-toggle--PaddingBlockStart);
+    padding-block-end: var(--#{$breadcrumb}__menu-toggle--PaddingBlockEnd);
     line-height: var(--#{$breadcrumb}__menu-toggle--c-menu-toggle--LineHeight);
   }
```
Comment (mcoker) [inline src/patternfly/components/Breadcrumb/breadcrumb.scss:124]: discussed in the working section that we can keep the padding as long as it it doesn't change the shape of the outer breadcrumb box. also looks like `line-height` is the same with or without this declaration, so I think we can nix this whole block!

**Review by mcoker — CHANGES_REQUESTED** [review]
Just one small nit!

**Review by mcoker — APPROVED** [review]

**Pull request merged**